File: SpringCloudDev/springcloud-vue-ui/src/entitys/integral/clientblacklist.js
                                    export default function area() {
  return {
    current: 1,
    size: 20,
    isNewRecord: false,
    query: {
    // 黑名单编号
  blackId: undefined,
      // 规则名称
  clientId: undefined,
      // 客户姓名
  clientName: undefined,
      // 开户行名称
  bankName: undefined,
      // 性别
  sex: undefined,
      // 联系电话
  mobile: undefined,
      // 预留字段一
  reserveColumn1: undefined,
      // 预留字段二
  reserveColumn2: undefined,
      // 预留字段三
  reserveColumn3: undefined,
          },
    rules: {
        blackId: [{
  required: true,
  message: '请输入黑名单编号',
  trigger: 'blur'
  }],
          clientId: [{
  required: true,
  message: '请输入规则名称',
  trigger: 'blur'
  }],
          clientName: [{
  required: true,
  message: '请输入客户姓名',
  trigger: 'blur'
  }],
          bankName: [{
  required: true,
  message: '请输入开户行名称',
  trigger: 'blur'
  }],
          sex: [{
  required: true,
  message: '请输入性别',
  trigger: 'blur'
  }],
          mobile: [{
  required: true,
  message: '请输入联系电话',
  trigger: 'blur'
  }],
          reserveColumn1: [{
  required: true,
  message: '请输入预留字段一',
  trigger: 'blur'
  }],
          reserveColumn2: [{
  required: true,
  message: '请输入预留字段二',
  trigger: 'blur'
  }],
          reserveColumn3: [{
  required: true,
  message: '请输入预留字段三',
  trigger: 'blur'
  }],
      }
  }
}

File: SpringCloudDev/springcloud-business-api/springcloud-integral-api/src/main/java/org/springclouddev/integral/entity/ClientBlackList.java
                                                                        package org.springclouddev.integral.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import com.baomidou.mybatisplus.annotation.TableId;
import org.springclouddev.core.mp.base.BaseEntity;
import com.baomidou.mybatisplus.annotation.TableField;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;
import com.fasterxml.jackson.databind.ser.std.ToStringSerializer;
/**
 * 实体类
 *
 * @author zhaobohao
 * @since 2020-01-15
 */
@Data
@Accessors(chain = true)
@TableName("tbl_client_black_list")
@EqualsAndHashCode(callSuper = true)
@ApiModel(value = "ClientBlackList对象", description = "ClientBlackList对象")
public class ClientBlackList extends BaseEntity  {

private static final long serialVersionUID = 1L;

                                    /**
         * 黑名单编号
         */
                        @ApiModelProperty(value = "黑名单编号")
                                                            @TableId("BLACK_ID")
                                                                                  private Long id;
                                        /**
         * 规则名称
         */
                        @ApiModelProperty(value = "规则名称")
                                    @TableField("CLIENT_ID")
                                                  private Long clientId;
                                        /**
         * 客户姓名
         */
                        @ApiModelProperty(value = "客户姓名")
                                    @TableField("CLIENT_NAME")
                                                  private String clientName;
                                        /**
         * 开户行名称
         */
                        @ApiModelProperty(value = "开户行名称")
                                    @TableField("BANK_NAME")
                                                  private String bankName;
                                        /**
         * 性别
         */
                        @ApiModelProperty(value = "性别")
                                    @TableField("SEX")
                                                  private String sex;
                                        /**
         * 联系电话
         */
                        @ApiModelProperty(value = "联系电话")
                                    @TableField("MOBILE")
                                                  private String mobile;
                                        /**
         * 预留字段一
         */
                        @ApiModelProperty(value = "预留字段一")
                                    @TableField("RESERVE_COLUMN1")
                                                  private String reserveColumn1;
                                        /**
         * 预留字段二
         */
                        @ApiModelProperty(value = "预留字段二")
                                    @TableField("RESERVE_COLUMN2")
                                                  private String reserveColumn2;
                                        /**
         * 预留字段三
         */
                        @ApiModelProperty(value = "预留字段三")
                                    @TableField("RESERVE_COLUMN3")
                                                  private String reserveColumn3;
            

    }
